### PR TITLE
LPS-164311 Testray Icons update

### PR DESCRIPTION
**Pull request by welitonpe**

### CODE REVIEW

**Comment by welitonpe** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/styles/components/layout/_sidebar.scss [hunk 1]
```diff
@@ -139,12 +143,20 @@ $normal-transition: opacity 1s ease;
...
 	transform: rotateY(0deg);
 	transition: transform 0.5s ease 0.4s;
 
 	&:hover {
 		background-color: #303376;
+		color: #fff;
+		fill: #fff;
+		transition: transform 0.5s ease 0.4s;
+	}
+	&:hover > svg path {
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/styles/components/layout/_sidebar.scss:156]: Space

```suggestion

	&:hover > svg path {
```

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/Icons/TestrayIcon.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+	orderFormCog,
+	page,
+	pencil,
+	plus,
+	plusUser,
+	polls,
+	print,
+	spanner,
+	trash,
+} from './spritemap';
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/Icons/TestrayIcon.tsx:32]: Export Once: 

import Spritemap from './spritemap';

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/Icons/TestrayIcon.tsx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+		'order-form-cog': orderFormCog,
+		page,
+		pencil,
+		plus,
+		'plus-user': plusUser,
+		polls,
+		print,
+		spanner,
+		trash,
+	};
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/Icons/TestrayIcon.tsx:64]: Move to spritemap

```suggestion
	const icon: any = {
		cog,
		community,
		display,
		drop,
		filter,
		'list-ul': listUl,
		merge,
		'order-form-cog': orderFormCog,
		page,
		pencil,
		plus,
		'plus-user': plusUser,
		polls,
		print,
		spanner,
		trash,
	};
```

**Review by kevenleone — CHANGES_REQUESTED** [review]

**Comment by welitonpe** [comment]
ci:test:sf

**Comment by welitonpe** [comment]
ci:forward

**Pull request closed without merging**